### PR TITLE
PowerShell: Update requirement to version 5

### LINKED ISSUE #3325: Would scoop update its minimum recommended PowerShell version to 5?
https://github.com/lukesampson/scoop/issues/2402, https://github.com/lukesampson/scoop-extras/pull/1993#issuecomment-482459556

Would scoop update to PS 5? Appveyor CI check only check 5 and 6, and 5 has several useful new cmdlet, e.g. `Expand-Archive`.

Windows 7 has PS 2, and 8 has 3, 8.1 has 4, 10 has 5, and I think the major Windows version that user has should be 10 and 7. For Windows 10, no update is needed, for Windows 7, there is no difference to update to 3 or 5.

### PR DESCRIPTION
**Pull request by niheaven**
Close #3325

### CODE REVIEW

**Comment by rikuben** [comment]
How about suggesting to install pwsh too as an alternative?

File: README.md [hunk 1]
```diff
@@ -57,13 +58,14 @@ If you've built software that you'd like others to use, Scoop is an alternative
 
 ## Requirements
 
-* Windows 7 SP1+ / Windows Server 2008+
-* [PowerShell 3](https://www.microsoft.com/en-us/download/details.aspx?id=34595) (or later) and [.NET Framework 4.5+](https://www.microsoft.com/net/download)
-* PowerShell must be enabled for your user account e.g. `set-executionpolicy remotesigned -s currentuser`
+- Windows 7 SP1+ / Windows Server 2008+
+- [PowerShell 5](http://aka.ms/wmf5download) (or later) and [.NET Framework 4.5](https://www.microsoft.com/net/download) (or later)
```
Comment (r15ch13) [inline README.md]:
```suggestion
- [PowerShell 5](https://aka.ms/wmf5download) (or later) and [.NET Framework 4.5](https://www.microsoft.com/net/download) (or later)
```

**Comment by niheaven** [comment]
Good idea! I'll add it later.

**Comment by Ash258** [comment]
Suggesting pwsh is nice, but how would updates be handled?

Problem which i am thinkig of today: When you install pwsh using scoop and you need to update pwsh. How do you do this? Because uninstallation will try to remove pwsh (uninstall old) and fail due to it is use.

**Comment by niheaven** [comment]
I don't suggest install pwsh using scoop, since it will not be registered in the system, and won't be found by many apps.

And in readme of scoop, pwsh is a prerequisite of installation, new user shouldn't install it by scoop, so it's ok, I think.

**Pull request merged**